backend/routes/userRoutes.js

```diff
@@ -160,6 +155,32 @@ router.get('/recommendedEvents/:userId', async (req, res) => {
     }
   })
   // ------------------------------------------------
+  // Get the id's of the events the user has commented on & how many comments the user has
+  const userComments = await prisma.comment.findMany({
+    where: { 
+      userPosting: user.username
+    }
+  })
+  let totalComments = 0
```

Comment:
totalComments can be a constant and instantiated by userComments length

Reply:
Done